docs/tutorials/chaos/slo-validation-chaos.md

```diff
@@ -0,0 +1,177 @@
+# Chaos Injection with SLOs
+
+Perform a joint Iter8 and [LitmusChaos](https://litmuschaos.io/) experiment. This joint experiment enables you to verify if an app continues to be resilient (satisfies SLOs) in the midst of chaos (pod kill).
+
+In the tutorial, the app consists of a Kubernetes service and deployment. The chaos experiment kills the app's pods intermittently. At the same time, the Iter8 experiment performs a load test of the app and validates its [service-level objectives (SLOs)](../../getting-started/concepts.md#service-level-objectives). 
+
+
+![Chaos with SLO Validation](images/slo-validation-chaos.png)
+
+???+ warning "Before you begin"
+    1. Try [your first experiment](../../getting-started/your-first-experiment.md). Understand the main [concepts](../../getting-started/concepts.md) behind Iter8 experiments.
+    2. Ensure that you have the [kubectl](https://kubernetes.io/docs/reference/kubectl/) CLI.
+    3. Install [Litmus](https://litmuschaos.io/) in Kubernetes using [these steps](https://docs.litmuschaos.io/docs/getting-started/installation).
+    4. Create the `httpbin` deployment file.
+    ```shell
+    cat <<EOF> deploy.yaml
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: httpbin
+      labels:
+        app: httpbin
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: httpbin
+      template:
+        metadata:
+          labels:
+            app: httpbin
+        spec:
+          containers:
+          - name: httpbin
+            image: kennethreitz/httpbin
+            ports:
+            - containerPort: 80
+          initContainers:
+          - name: init-myservice
+            image: busybox:1.28
+            command: ['sh', '-c', 'sleep 1']
```

Comment:
This container seems to be the difference between this and the way we start _httpbin_ in the other experiments (`kubectl create deploy httpbin --image=kennethreitz/httpbin --port=80`). Is this addition necessary? Does it need explanation especially after being told to try _your first experiment_

Reply:
The deployment is structured purely for moving the demo narrative forward ... 

We want SLOs to fail when scale = 1, and we want SLOs to succeed when scale = 3.

Structuring the deployment as shown in this example seems to achieve this effect predictably well.

***

I don't think we need to explain the above for the end-user, since that is irrelevant to the purpose of the tutorial (which is here's how you combine chaos and Iter8 experiments to test your app).

Reply:
More important, the user is expected to replace the `httpbin` deployment in this tutorial, with their own app (and I believe, the explanations under expandable sections provide enough details for the user to accomplish this).